infra/scoped/secrets.tf

```diff
@@ -26,6 +26,31 @@ data "aws_secretsmanager_secret_version" "openathens_callback_url" {
   secret_id = aws_secretsmanager_secret.openathens_callback_url.id
 }
 
+# auth0 action configuration
+resource "aws_secretsmanager_secret" "redirect_action_url" {
+  name = "identity/${terraform.workspace}/redirect_action_url"
+}
+
+data "aws_secretsmanager_secret_version" "redirect_action_url" {
+  secret_id = aws_secretsmanager_secret.redirect_action_url.id
+}
+
+resource "aws_secretsmanager_secret" "redirect_action_url_stage" {
+  name = "identity/${terraform.workspace}/redirect_action_url_stage"
+}
```

Comment:
Think this might've been left in by accident but you don't need it because `terraform.workspace` is doing the environment separation for you